rootful_network.py

```diff
@@ -565,22 +565,27 @@ def setup(args, con):
         "container_id": container.id,
         "container_name": container.name,
         "port_mappings": portmappings(args.publish),
-        "networks": {
-            network.name: {
+        "networks": {},
+        "network_info": {},
+    }
+
+    interface_increment = 0
+    for podman_network in args.networks:
```

Comment:
A typical more pythonic way to do this is to use `for interface_increment, podman_network in enumerate(args.networks)`. See also https://docs.python.org/3/library/functions.html#enumerate.